cmd/promoted-image-governor/README.md

```diff
@@ -0,0 +1,53 @@
+# promoted-image-governor
+
+## What it does
+
+`promoted-image-governor` is a tool with the following features:
+
+- For any image stream on every build-farm cluster, if it contains a tag promoted by some ci-operator's configuration, then
+  every tag in the stream has to be promoted too. Any violated tag will be deleted by this tool.
+- Generate [the image mapping files](https://github.com/openshift/release/tree/master/core-services/image-mirroring/openshift) for OpenShift/origin.
+- Explain why an `imagestreamtag` exists.
+
+
+## Why it exists
+
+- Regulate the image streams with promoted tags.
```

Comment:
I'm not sure about the "regulate" here. Regulate towards what goal? I think the main purpose is to delete the stale images that were promoted in the past but are no more, no?

Reply:
Updated.